test/e2e/broker_trigger_test.go

```diff
@@ -268,3 +270,110 @@ func TestBrokerWithConfig(t *testing.T) {
 		}
 	})
 }
+
+func TestBrokerExternalTopicTrigger(t *testing.T) {
+	testingpkg.RunMultiple(t, func(t *testing.T) {
```

Comment:
Can we use the new framework for new tests?

Reply:
In addition, this test doesn't prevent regression in the feature since I can just drop the new reconciler logic and it will still work, what about using a custom role and the permissions to only publish and subscribe to a particular topic?

Reply:
> > Can we use the new framework for new tests?
> 
> you mean the reconciler-test? I decided against that since I'd have to extend/customize the broker, since the upstream template doe not really allow (n) custom annotations.

we can always create our own Kafka broker template